tests/test_cassandra.py

```diff
@@ -68,9 +70,8 @@ def test_cassandra_execute_exception(monkeypatch):
 
     monkeypatch.setattr('zmon_worker_monitor.builtins.plugins.cassandra_wrapper.Cluster', cluster)
 
-    cassandra = CassandraWrapper(node, keyspace)
-
     with pytest.raises(RuntimeError):
+        cassandra = CassandraWrapper(node, keyspace)
```

Comment:
We need to cover another test case with exception in ``execute`` rather than ``connect``.